18F/identity-idp
# Use Icon List component for requested attributes consent

## 🛠 Summary of changes

Updates consent screens to use the [design system Icon List component](https://designsystem.digital.gov/components/icon-list/).

**Why?**

- Standardize on design system components
- Reduce size of application stylesheet by removing ad-hoc `success-bullets` styles

Recommended to review code changes with whitespace revisions hidden: [identity-idp/pull/9555/files?w=1](https://github.com/18F/identity-idp/pull/9555/files?w=1)

## 📜 Testing Plan

1. Start [sample app](https://github.com/18F/identity-oidc-sinatra) in a separate process
2. Prerequisite: If you've already consented with the sample app, create a new account or revoke consent from the account dashboard "Your connected accounts" screen
3. Go to http://localhost:9292
4. Click "Sign in"
5. Sign in
6. Observe new consent screen
7. Go to http://localhost:3000/user_authorization_confirmation
   8. Typically you'd see this if you're signing in to a different partner application that you had previously consented to while already being signed in
9. Observe new consent screen

## 👀 Screenshots

Screen|Before|After
---|---|---
Consent|![consent-before](https://github.com/18F/identity-idp/assets/1779930/f6058e5e-f550-4d3d-b83b-a6a96a856cf4)|![consent-after](https://github.com/18F/identity-idp/assets/1779930/e9115b29-a292-4f6e-812d-21df63a2334f)
Authorization Confirmation|![signed-in-before](https://github.com/18F/identity-idp/assets/1779930/348a600c-6b8a-4e5d-bcd8-379d67c21982)|![signed-in-after](https://github.com/18F/identity-idp/assets/1779930/de385495-2ca2-4b43-bc35-1e7241fb9a4e)

It's faked with duplicate "Email address", but for extra assurance, here's what the screen will look like with multiple requested attributes:

![localhost_3000_sign_up_completed(Computer Standard)](https://github.com/18F/identity-idp/assets/1779930/b9918503-8798-4c06-9e54-c6588f1f1134)
